mobile/voice_journal_app/lib/Pages/Alec_Recording.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
```

Comment:
I don't think we need another page as a starting point for the app. Let's remove this page if there's no need for it.